tests/console/btrfs_qgroups.pm

```diff
@@ -65,6 +65,14 @@ sub run() {
     }
     assert_script_run "cp nofile c/nofile";
 
+    # Check for bsc#993841
+    assert_script_run "btrfs subvolume create e";
+    assert_script_run "btrfs qgroup limit 50m e .";
+    assert_script_run "! for c in {1..40}; do dd if=/dev/zero bs=1M count=40 of=e/file; done";
+    script_run "sync";
+    assert_script_run "rm e/file";
+    assert_script_run "touch e/newfile";
```

Comment:
Fixed with fail_message => 'bsc#993841'. Test is for qgroup limits, which doesn't work properly - I do not think we should treat this as soft-fail